Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -277,3 +282,24 @@ private func verify(payload: MailgunForwardPayload) -> Bool {
   )
   return payload.signature == digest
 }
+
+extension PartialIso where A == (String?, Int?), B == Pricing {
+  fileprivate static var pricing: PartialIso {
+    return PartialIso(
+      apply: { plan, quantity in
+        let pricing: Pricing
+        if let quantity = quantity {
+          pricing = .team(quantity)
+        } else if let plan = plan, let billing = Pricing.Billing(rawValue: plan) {
+          pricing = .individual(billing)
+        } else {
+          pricing = .default
```

Comment:
I moved this logic from the site middleware to here.

now, I would have expected that this `.default` would not be necessary but it is :/ I need to dig into this more, but going to leave for now